src/VisualStudio/TestUtilities2/ProjectSystemShim/Framework/MockVsFileChangeEx.vb

```diff
@@ -172,6 +172,14 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Fr
             Return Task.CompletedTask
         End Function
 
+        Public Function UnadviseFileChangesAsync(cookies As IReadOnlyCollection(Of UInteger), Optional cancellationToken As CancellationToken = Nothing) As Task(Of String()) Implements IVsAsyncFileChangeEx.UnadviseFileChangesAsync
```

Comment:
Q: are these result of package version changes or something else?

Reply:
I got build errors that these two functions are not implemented in `IVsAsyncFileChangeEx`. So I added them.